lib/utils.js

```diff
@@ -9,8 +9,10 @@ exports.defineCachedRelations = defineCachedRelations;
 exports.sortObjectsByIds = sortObjectsByIds;
 exports.setScopeValuesFromWhere = setScopeValuesFromWhere;
 exports.mergeQuery = mergeQuery;
+exports.createPromiseCallback = createPromiseCallback;
 
 var traverse = require('traverse');
+var Promise = require('bluebird');
```

Comment:
#449 was landed, please rebase on top of the current master and remove the infrastructure changes that are no longer needed.